data/sea/54-zebra.h

```diff
@@ -154,68 +185,142 @@ static ierror_msg_t zebra_translate
     return 0;
 }
 
-static int64_t zebra_translate_table
+/* Translate Zebra enum tags to Icicle bool flags, e.g. is_some, is_right, etc.
+   We only allow at most two variants since Icicle's runtime representation
+   of enums is unsuitable for more.
+ */
+static uint64_t zebra_translate_column_tags
+    ( const int64_t variant_count
+    , void **dst
+    , const int64_t *src )
+{
+    assert (variant_count <= 2);
```

Comment:
I just worry it will get forgotten, and someone will get a cryptic error in production.

I wasn't thinking of anything too rigorous, just something like this:
```c
if (variant_count > 2) {
    fprintf (stderr, "Fatal error: found an enum in the zebra input with more than two variants (it had %" PRId64 " variants)\n", variant_count);
    exit (1);
}
```